### PR TITLE
chore: Repo clean up code snippet that are migrated to python-docs-samples

### PR DESCRIPTION
**Pull request by msampathkumar**
Repo clean up! Code samples are migrated to python-docs-samples.

For more details, check http://b/257074671

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-tasks/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by msampathkumar** [comment]
A CL for updating docs is in WIP! Added don't merge label for now.

**Comment by dandhlee** [comment]
`feat`, `fix` or `docs` prefix on the PR title triggers a new release on the client library, FYI.

File: samples/CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,3 @@
+For add manually written code, please check https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud_tasks/snippets
```
Comment (dandhlee) [inline samples/CONTRIBUTING.md:1]:
```suggestion
For hand written samples, please check https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud_tasks/snippets
```

File: samples/snippets/README.md [hunk 2]
```diff
@@ -1,87 +0,0 @@
-# Google Cloud Tasks Samples
```
Comment (dandhlee) [inline samples/snippets/README.md:1]: Could you replace this with something like https://github.com/googleapis/python-kms/tree/main/samples/snippets

**Review by dandhlee — CHANGES_REQUESTED** [review]

File: samples/snippets/README.md (hunk 2 above)
Comment (msampathkumar) [reply samples/snippets/README.md:1]: Thank you for the reference @dandhlee

**Comment by dandhlee** [comment]
Please request for a review once snippet-bot check is passing.

**Comment by msampathkumar** [comment]
cl/489962491 and cl/492048376 are completed. Rerunning snippet bot checker now!

**Review by dandhlee — COMMENTED** [review]
Seems like we're deleting more files than we've migrated, can you confirm this is intentional?

File: samples/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Samples migrated
+================
+
+New location: https://github.com/GoogleCloudPlatform/python-docs-samples/tree/main/cloud_tasks/snippets
```
Comment (dandhlee) [inline samples/README.md:4]: We still have generated samples. Could you move this down to `samples/snippets/README.md`?

**Review by dandhlee — APPROVED** [review]
LGTM, one minor nit.

**Pull request merged**